superset/views/core.py

```diff
@@ -372,6 +372,7 @@ def form_post(self, form):
             flash(
                 message,
                 'danger')
+            stats_logger.incr('failed_csv_upload')
```

Comment:
Is there merit in including the `sqllab.` prefix? Note I'm not sure whether historically we use `sqllab` or `sql_lab`.

Reply:
per our discussion I think the current state is fine.